kroxylicious-operator/src/main/resources/META-INF/fabric8/kafka-proxies.kroxylicious.io-v1.yml

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
Note to self... this is a copy of the CRD which the JOSDK test support seems to expect to live here. We should avoid this duplication with the CRD in `src/main/resources/kubernetes/Crd-KafkaProcy.yaml`. So either make this one canonical or the other one.